--- quit.h	2014-11-24 18:12:39.773375352 -0800
+++ patch-files/quit.h	2014-11-24 18:13:13.354394640 -0800
@@ -23,16 +23,16 @@
 #if !defined (octave_quit_h)
 #define octave_quit_h 1
 
-#ifdef __cplusplus
-#include <new>
-extern "C" {
-#endif
-
 #include <stdio.h>
 
 #include <signal.h>
 #include <setjmp.h>
 
+#ifdef __cplusplus
+#include <new>
+extern "C" {
+#endif
+
 #if defined (__WIN32__) && ! defined (_POSIX_VERSION)
 
 #define WIN32_LEAN_AND_MEAN
